assets/app/styles/_create.less

```diff
@@ -0,0 +1,225 @@
+//
+// Create App
```

Comment:
Next time we break up a file like this, we should probably do it as a separate, smaller PR. It's too easy to drop changes on rebase otherwise.